docs/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+Closes #
+
+Changes proposed in this pull request:
+
+-
+-
+-
+
+Pull request checklist:
+
+- [ ] branch is based on `master`
+- [ ] proper commit messages
+- [ ] code is properly tested
+- [ ] documentation is updated
```

Comment:
```suggestion
- [ ] documentation updated
```

Neither is grammatically right or wrong, but brevity is valued in English style.